demisto_sdk/commands/validate/validators/BA_validators/BA106_is_from_version_sufficient_indicator_field.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from packaging.version import Version
+
+from demisto_sdk.commands.content_graph.objects.indicator_field import IndicatorField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+from demisto_sdk.commands.validate.validators.super_classes.BA106_is_from_version_sufficient import (
+    IsFromVersionSufficientValidator,
+)
+
+INDICATOR_FIELD_TYPE_TO_MIN_VERSION = {
+    "html": "6.1.0",
+    "grid": "5.5.0",
+}
+
+ContentTypes = IndicatorField
+
+
+class IsFromVersionSufficientIndicatorFieldValidator(
+    IsFromVersionSufficientValidator, BaseValidator[ContentTypes]
+):
+    description = (
+        "Validate that the indicator fromversion is sufficient according to its type"
+    )
+    error_message = "The fromversion of IndicatorField with type {0} must be at least {1}, current is {2}."
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        return [
+            ValidationResult(
```

Comment:
have the is_valid function create the ValidationResults, but call to a helper function to do the logic